azure-quantum/azure/quantum/qiskit/backends/ionq.py

```diff
@@ -348,6 +383,39 @@ def __init__(self, name: str, provider: "AzureQuantumProvider", **kwargs):
         super().__init__(configuration=configuration, provider=provider, **kwargs)
 
 
+class IonQForteBackend(IonQBackend):
+    backend_names = ("ionq.qpu.forte-1",)
+
+    def __init__(self, name: str, provider: "AzureQuantumProvider", **kwargs):
+        """Base class for interfacing with an IonQ Forte QPU backend"""
+        gateset = kwargs.pop("gateset", "qis")
+        default_config = BackendConfiguration.from_dict(
+            {
+                "backend_name": name,
+                "backend_version": __version__,
+                "simulator": False,
+                "local": False,
+                "coupling_map": None,
+                "description": "IonQ Aria QPU on Azure Quantum",
```

Comment:
Could you update this description
                "description": "IonQ Aria QPU on Azure Quantum",

Reply:
Fixed